server/schemas/integration.graphql

```diff
@@ -0,0 +1,56 @@
+enum IntegrationType {
+  Public
+  Private
+}
+
+type Integration implements Node {
+  id: ID!
+  name: String!
+  description: String
+  logoUrl: URL!
+  type: IntegrationType!
+  token: String!
+  developerId: ID!
+  developer: User!
+  webhook: [Webhook!]!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
+
+# Inputs
+input CreateIntegrationInput {
+  name: String!
+  description: String
+  logoUrl: URL!
+  type: IntegrationType!
+}
+
+input UpdateIntegrationInput {
+  integrationId: ID!
+  name: String
+  description: String
+  logoUrl: URL
+}
+
+input DeleteIntegrationInput {
+  integrationId: ID!
+}
+
+# Payload
+type IntegrationPayload {
+  integration: Integration!
+}
+
+type DeleteIntegrationPayload {
+  integrationId: ID!
+}
+
+extend type Query {
+  integrations(type: IntegrationType, keyword: String): [IntegrationPayload]!
+}
+
+extend type Mutation {
+  createIntegration(input: CreateIntegrationInput!): IntegrationPayload
+  updateIntegration(input: UpdateIntegrationInput!): IntegrationPayload
+  deleteIntegration(input: DeleteIntegrationInput!): DeleteIntegrationPayload
```

Comment:
To install an integration to the workspace, just we can invite it as a member of the workspace. So there is no need to add a new mutation to install it. Is it right?

Reply:
Yes sure